lighthouse-core/gather/driver/network-monitor.js

```diff
@@ -49,14 +61,29 @@ class NetworkMonitor extends EventEmitter {
     this.off = (event, listener) => super.off(event, listener);
   }
 
+  /**
+   * @param {{target: {targetId: string}, session: LH.Gatherer.FRProtocolSession}} session
+   */
+  async _onTargetAttached({session, target}) {
+    const targetId = target.targetId;
+    if (this._sessions.has(targetId)) return;
```

Comment:
won't the targetManager already be ensuring this won't be called twice with the same targetId?

Reply:
yes in this version I don't believe this line is necessary anymore 👍